cynic-codegen/src/fragment_derive/fragment_impl.rs

```diff
@@ -372,7 +372,7 @@ impl quote::ToTokens for SpreadSelection {
         tokens.append_all(quote_spanned! { self.span =>
             <#field_type as cynic::QueryFragment>::query(
                 builder
-            )
+            );
```

Comment:
I think if you change this to:

```
<#field_type as cynic::QueryFragment>::query(
    builder
        .inline_fragment()
        .select_children::<<#field_type as cynic::QueryFragment>::VariablesFields>()
)
```

It should allow us to support multiple spreads per `QueryFragment`.